### PR TITLE
Deprecate formula

### PR DESCRIPTION
**Pull request by derekparker**
The formula is not maintained and is no longer necessary as the
installation on OSX is just as simple as any other OS now.

### CODE REVIEW

**Comment by derekparker** [comment]
After this is merged I will archive the repo.

**Comment by derekparker** [comment]
cc @aarzilli

**Review by aarzilli — APPROVED** [review]
LGTM, but I never had merge permissions for this repo.

**Pull request merged**
